src/s3/client.rs

```diff
@@ -2355,6 +2361,76 @@ impl<'a> Client<'a> {
         })
     }
 
+    pub async fn set_bucket_quota(
+        &self,
+        args: &SetBucketQuotaArgs<'_>,
+    ) -> Result<SetBucketQuotaResponse, Error> {
+        let mut headers = Multimap::new();
+        if let Some(v) = &args.extra_headers {
+            merge(&mut headers, v);
+        }
+        let mut query_params = Multimap::new();
+        query_params.insert("bucket".into(), args.bucket_name.into());
+
+        let mut query_params = Multimap::new();
+        query_params.insert("bucket".into(), args.bucket_name.into());
+
+        let data = serde_json::to_string(&args.quota)?;
+
+        let resp = self
+            .execute(
+                Method::PUT,
+                &"us-east-1".into(),
+                &mut headers,
+                &query_params,
+                "minio/admin/v3/set-bucket-quota".into(),
```

Comment:
You should keep Minio admn APIs as separate package like [minio-java](https://github.com/minio/minio-java/tree/master/adminapi/src) or [minio-py](https://github.com/minio/minio-py/blob/master/minio/minioadmin.py); not inside `s3`.

Reply:
Maybe just move it to the separate module?

Reply:
like `src/admin` or `src/adminapi`

Reply:
Yep. That's what I mean